boards/native/include/board.h

```diff
@@ -26,6 +26,10 @@
 
 #include <stdint.h>
 
+#ifndef BOARD_NATIVE
+#define BOARD_NATIVE
+#endif
```

Comment:
Why? `BOARD_NATIVE` is defined globally when compiling for `native`, so no need for that definition

Reply:
You're right, I've removed this.